python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -302,11 +302,12 @@
 # %% [markdown]
 # ## Model evaluation using cross-validation
 #
-# In the previous example, we split the original data into a training set and a
-# testing set. This strategy has several issues: in a setting where the amount
-# of data is small, the subset used to train or test will be small. Besides, a
-# single split does not give information regarding the confidence of the
-# results obtained.
+# In the previous example, we split the original data into a training set and a 
+# testing set. The score of a model will in general depend on the way we make 
+# such a split. One downside of doing a single split is that it does not give
```

Comment:
```suggestion
# such a split. Doing a single split does not give
```

Reply:
Hmmm I think it is good enough as it is but thanks for the suggestion 😉